st2.json

```
@@ -1,137 +1,152 @@
 {
   "variables": {
-    "purpose": "Packer template for building StackStorm Vagrant '.box' & '.ova' Virtual Appliance",
-    "st2_version": "{{env `ST2_VERSION`}}",
     "box_version": "{{env `BOX_VERSION`}}",
-    "st2_user": "st2admin",
+    "purpose": "Packer template for building StackStorm Vagrant '.box' \u0026 '.ova' Virtual Appliance",
     "st2_password": "Ch@ngeMe",
-    "vm_name": "st2",
-    "vm_description": "Box with StackStorm (aka 'IFTTT for Ops') event-driven automation platform: auto-remediation, security responses, facilitated troubleshooting, complex deployments, ChatOps and more. \n* https://stackstorm.com/ \n* Documentation: https://docs.stackstorm.com/ \n* Community: https://stackstorm.com/community-signup \n* Forum: https://forum.stackstorm.com/"
+    "st2_user": "st2admin",
+    "st2_version": "{{env `ST2_VERSION`}}",
+    "vm_description": "Box with StackStorm (aka 'IFTTT for Ops') event-driven automation platform: auto-remediation, security responses, facilitated troubleshooting, complex deployments, ChatOps and more. \n* https://stackstorm.com/ \n* Documentation: https://docs.stackstorm.com/ \n* Community: https://stackstorm.com/community-signup \n* Discussions: https://github.com/StackStorm/st2/discussions",
+    "vm_name": "st2"
   },
+  "builders": [
+    {
+      "boot_command": [
+        "<enter><wait><f6><wait><esc><wait>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs><bs><bs><bs><bs><bs><bs><bs>",
+        "<bs><bs><bs>",
+        "/install/vmlinuz",
+        " initrd=/install/initrd.gz",
+        " priority=critical",
+        " locale=en_US",
+        " file=/media/preseed.cfg",
+        "<enter>"
+      ],
+      "boot_wait": "10s",
+      "disk_size": 40960,
+      "export_opts": [
+        "--ovf10",
+        "--vsys",
+        "0",
+        "--version",
+        "{{user `st2_version`}}",
+        "--product",
+        "StackStorm",
+        "--producturl",
+        "https://stackstorm.com/",
+        "--description",
+        "{{user `vm_description`}}"
+      ],
```

Same as in the other comment, let's keep the original compact formatting for readability:
```suggestion
      "export_opts": [
        "--ovf10",
        "--vsys", "0",
        "--version", "{{user `st2_version`}}",
        "--product", "StackStorm",
        "--producturl", "https://stackstorm.com/",
        "--description", "{{user `vm_description`}}"
      ],
```